core/common/src/main/java/alluxio/wire/ConfigInfo.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.wire;
+
+import com.google.common.base.Objects;
+
+import java.io.Serializable;
+
+import javax.annotation.concurrent.NotThreadSafe;
+
+/**
+ * The Alluxio runtime configuration information.
+ */
+@NotThreadSafe
+public final class ConfigInfo implements Serializable {
```

Comment:
Perhaps call this `ConfigProperty`? Info is generic and doesn't show whether it's plural or not.

Reply:
Changed, thanks!
Better Name!